src/main.c

```diff
@@ -79,7 +85,12 @@ int main()
                 }
                 else if (event.key.keysym.sym == SDLK_RETURN)
                 {
-                    if (menu.selector == MENU_ITEM_SINGLE_PLAYER)
+                    if (menu.selector == MENU_ITEM_USER_MANUAL)
+                    {
+                        quit = user_manual(p_window);
```

Comment:
user_manual prototype is defined wrongly in include/menu.h can you please fix it as well. I couldn't compile it without fixing it.

Reply:
I don't get the problem you encountered, the project executes correctly on my end.